packages/console/src/types/index.tsx

```diff
@@ -138,3 +139,13 @@ export type SelectColor =
     | "error"
     | "info"
     | "warning";
+
+export type CheckboxSize = "small" | "medium";
+
+export type CheckboxColor =
+    | "primary"
+    | "secondary"
+    | "success"
+    | "error"
+    | "info"
+    | "warning";
```

Comment:
Can we merge all Color types into a single type? Can it be done for size too?

Reply:
Done